server/routes/eligibility.js

```diff
@@ -58,9 +49,7 @@ module.exports = ({ licenceService, nomisPushService }) => (router, audited, con
     asyncMiddleware(standard.callbackPost('suitability', checksPassedCallback))
   )
 
-  router.get('/excluded/:bookingId', asyncMiddleware(get))
```

Comment:
just checking this is intentional
So you don't need to treat excluded as a special case - you are treating all eligibilty routes in the same way?

Reply:
Yes. I'm using a copy of standard.js 'get'. Only change is adding the cja2003Sza flag for the views.  The 'suitability' form also needs this flag now because Louise added another option to this form: 'Offender not currently serving for a terrorism or terrorism-related offence but with a history of such offending'